File: taskManager/labfinal_210042111/StatisticsCollector.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace labfinal_210042111
{
    public class StatisticsCollector
    {
        public int CreateCommandCount { get; private set; }
        public int UpdateCommandCount { get; private set; }
        public int DeleteCommandCount { get; private set; }
        public int IssuesCreated { get; private set; }

        public void Increment(CommandType type)
        {
            switch (type)
            {
                case CommandType.create:
                    CreateCommandCount++;
                    break;
                case CommandType.update:
                    UpdateCommandCount++;
                    break;
                case CommandType.delete:
                    DeleteCommandCount++;
                    break;
            }
        }

        public void IncrementIssueCreated() => IssuesCreated++;

    }
}

File: TaskManager/210042141_TaskManager/src/StatisticsCollector.java
public class StatisticsCollector {
    public int createCommandCount;
    public int updateCommandCount;
    public int deleteCommandCount;

    public StatisticsCollector(int createCommandCount, int updateCommandCount, int deleteCommandCount){
        this.createCommandCount = createCommandCount;
        this.updateCommandCount = updateCommandCount;
        this.deleteCommandCount = deleteCommandCount;
    }

    public void increaseCreateCommand(){
        createCommandCount++;
    }

    public void increaseUpdateCommand(){
        updateCommandCount++;
    }

    public void increaseDeleteCommand(){
        deleteCommandCount++;
    }

    public int  getCreateCount(){
        return createCommandCount;
    }

    public int getUpdateCount(){
        return updateCommandCount;
    }

    public int getDeleteCount(){
            return deleteCommandCount;
    }



}